tethne/tests/test_models_lda.py

```diff
@@ -45,10 +45,11 @@ def test_mallet_to_phi_featureset(self):
         self.assertIsInstance(phi, FeatureSet)
         self.assertEqual(len(phi), 20)
 
-    def test_mallet_to_theta_featureset_tab(self):
-        from tethne import mallet_to_theta_featureset
-        theta = mallet_to_theta_featureset('tethne/tests/data/mallet/tutorial_doc_topic.txt')
-        self.assertIsInstance(theta, FeatureSet)
+    # TODO: Fix this test (TETHNE-147)
```

Comment:
Cool. I think that there is a compatibility issue with MALLET -- if I recall correctly, a new minor version recently changed their output format. Personally I'm reluctant to support the change until we're sure that they're not going to yank the rug out from under us again. We can discuss this further later on.